docs/content/contributing/2.docs/4.creating-new-docs.md

```diff
@@ -0,0 +1,82 @@
+# Adding Docs to Your Project
+
+Adding documentation helps developers work with your Vue Storefront integration. While you're more than welcome to use any docs tech stack you want, you may also want to use the Vue Storefront Docs theme to be consistent with the rest of the ecosystem. This is the Nuxt Layer used by all Vue Storefront maintained projects.
+
+
+::steps
+#step-1
+## Initialize Your Nuxt Project
+First, create a Nuxt 3 project inside of your project repo. From the root of your repo, run:
+
+```sh
+npx nuxi init docs
+```
+
+This creates a Nuxt 3 project inside a `docs` folder with the following structure:
+
+```
+├── README.md
+├── app.vue
+├── nuxt.config.ts
+├── package.json
+├── public
+│   └── favicon.ico
+├── server
+│   └── tsconfig.json
+├── tsconfig.json
+└── yarn.lock
+```
+
+Since the Docs Theme takes care of most of the design, logic, and behavior of the site - you should delete the `app.vue` to allow the theme to use its own UI.
```

Comment:
```suggestion
Since the Docs Theme takes care of most of the design, logic, and behavior of the site, you should delete the `app.vue` to allow the theme to use its own UI.
```